src/components/homepage/SpotPreview.tsx

```diff
@@ -1,21 +1,45 @@
 import Image from "next/image";
+import type { StaticImageData } from "next/image";
+
+import RatingInfo from "./RatingInfo";
+import TagIcons from "./TagIcons";
 
 export default function SpotPreview({
   title,
-  // location,
+  location,
   imageLink,
+  totalReviews,
   averageRating,
+  tags,
 }: {
   title: string;
   location: string;
-  imageLink: string;
+  imageLink: StaticImageData;
+  totalReviews: number;
   averageRating: number;
+  tags: string[];
 }) {
   return (
-    <div>
-      <Image src={imageLink} alt={title} />
-      <div>{title}</div>
-      <div>{averageRating}</div>
+    <div className="m-[10px] flex h-[327px] w-[307px] cursor-pointer flex-col items-center rounded-[20px] bg-white p-3">
```

Comment:
we should really avoid using pixel values for heights and widths of anything especially if its a container - this could result in future problems when we try to make the site mobile friendly